components/textfield/stories/template.js

```diff
@@ -183,6 +190,51 @@ export const Template = ({
 			size: "s",
 			customClasses: customProgressCircleClasses,
 		}, context))}
+		${when(helpText, () => 
+				HelpText({
+					text: helpText,
+					variant: isInvalid ? "negative" : "neutral",
+					size,
+					hideIcon: true,
+					isDisabled
+				}, context ))}
 	</div>
 	`;
 };
+
+export const HelpTextOptions = (args, context) => Container({
+	direction: "column",
+	withBorder: false,
+	withHeading: false,
+	content: html`
+		${Container({
+			withBorder: false,
+			heading: "Description",
+			content: Template({...args, isRequired: true, labelText: "Username", value: "lisawilson24", helpText: "Username must be at least 8 characters."}, context),
+		})}
+		${Container({
+			withBorder: false,
+			heading: "Error message",
+			content: Template({...args, isRequired: true, labelText: "Email address", value: "abc@adobe.com", helpText: "Enter your email address", isInvalid: true }, context),
+		})}
+	`
+});
+
+export const HelpTextOptionsTextArea = (args, context) => Container({
```

Comment:
Looking at the other nested components (like meter and form), they have their own `component.template.js` file. Did we want to do the same with this? Then we could move `HelpTextOptionsTextArea` into a `textarea.template.js` file.

Reply:
Oh and then basically all of this `when()` would be the textarea `Template()` (just without any of the `multiline` stuff)

https://github.com/adobe/spectrum-css/blob/d22591967c9df4babb6901dd4d45448f6261e77f/components/textfield/stories/template.js#L155